samples/word/01-basics/search.yaml

```diff
@@ -7,54 +7,62 @@ api_set:
     WordApi: 1.1
 script:
     content: |-
-        $("#setup").click(setup);
-        $("#basic").click(basicSearch);
-        $("#wildcard").click(wildcardSearch);
-
-
-        function basicSearch() {
-            Word.run(function (context) {
-                var results = context.document.body.search("Online");
-                context.load(results);
-                return context.sync()
-                    .then(function () {
-                        //lets traverse the search results... and highlight..
-                        for (var i = 0; i < results.items.length; i++) {
-                            results.items[i].font.highlightColor = "yellow";
-                        }
-                        return context.sync();
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+        $("#setup").click(() => tryCatch(setup));
+        $("#basic-search").click(() => tryCatch(basicSearch));
+        $("#wildcard-search").click(() => tryCatch(wildcardSearch));
+
+        async function basicSearch() {
+            await Word.run(async (context) => {
+                let results = context.document.body.search("Online");
+                results.load("font/highlightColor");
+
+                await context.sync();
+
+                //lets traverse the search results... and highlight..
+                for (let i = 0; i < results.items.length; i++) {
+                    results.items[i].font.highlightColor = "yellow";
+                }
+
+                await context.sync();
+            });
         }
 
-        function wildcardSearch() {
-            Word.run(function (context) {
+        async function wildcardSearch() {
+            await Word.run(async (context) => {
                 // check out how wildcard expression are built, also use the second parameter of the search method to include search modes (i.e. we are using wildcards).
-                var results = context.document.body.search("$*.[0-9][0-9]", { matchWildcards: true });
-                context.load(results);
-                return context.sync()
-                    .then(function () {
-                        //lets traverse the search results... and highlight..
-                        for (var i = 0; i < results.items.length; i++) {
-                            results.items[i].font.highlightColor = "red";
-                            results.items[i].font.color = "white";
-                        }
-                        return context.sync();
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+                let results = context.document.body.search("$*.[0-9][0-9]", { matchWildcards: true });
+                results.load("font/highlightColor,font/color");
+
+                await context.sync();
+
+                //lets traverse the search results... and highlight..
```

Comment:
Another place where "lets" should be "Let's"

Reply:
local fixed